doc/modules/ROOT/pages/end-to-end-examples/load-data-via-graph-construction.adoc

```diff
@@ -0,0 +1,167 @@
+= Load data to a projected graph via graph construction
+
+[TIP]
+====
+Jupyter Notebook available on https://github.com/neo4j/graph-data-science-client/blob/main/examples/load-data-via-graph-construction.ipynb[Github^]
+====
+
+This notebook shows the usage of the `gds.alpha.graph.construct` method
+(available only in GDS 2.1+) to build a graph directly in memory.
+
+*NOTE:* If you are using AuraDS, it is currently not possible to write
+the projected graph back to Neo4j.
+
+== Setup
```

Comment:
General nitpick: Our asciidoc code style is to have to blank lines before a new header (https://github.com/neo4j/graph-data-science-client/tree/main/doc#authoring)

Reply:
Sure thing.